### DISCUSSION TITLE
PR Run Test : ERROR please Help

### DISCUSSION THREAD

**Discussion by Mersag** [neuefische/questions#76] [Web] [answered]
Hallo ,

ich habe meinen ersten **PR** ausgeführt und infolgedessen auch ein **Run Test**. 
Ich erhalte diese Fehlermeldung und kann es nicht beheben.

![image](https://user-images.githubusercontent.com/107836739/185076342-5b1bd17f-25cc-43c7-829b-d6a0f074f6e1.png)
### Ziel : 
Fehlermeldung auf Git Hub PR beheben.

**Comment by PReichetanz** [comment, marked as answer]
Hallo,

wichtig ist für uns zunächst [ein Link zum PR](https://github.com/Mersag/capstone-projekt/pull/3), in dem der Fehler auftritt; in diesem Fall auch ein [Link zur Fehlermeldung selbst](https://github.com/Mersag/capstone-projekt/runs/7874825783?check_suite_focus=true):

```
FAIL src/components/Button/Button.spec.js
  ● Test suite failed to run

    Cannot find module './' from 'src/components/Button/Button.spec.js'

       6 | import userEvent from '@testing-library/user-event';
       7 |
    >  8 | import Button from './';
         | ^
       9 |
      10 | describe('Button component', () => {
      11 | 	it('should render children', () => {
```

Der Fehler liegt im Import des Buttons, die Datei muss spezifiziert werden: `import Button from './Button.js'`.

Du kannst verhindern, dass ein solcher Fehler erst durch GitHub Actions gemeldet wird, indem Du beim Schreiben der Tests mit `npm run test:watch` bei jeder Änderung die Tests durchlaufen lässt.

**Reply by Mersag** [reply]
Hallo, alles klar. Erst der Link zur PR, dann der Link zur Fehlermeldung ... Da Button eine bereits im Template vorhanden Komponente ist (war), habe ich diese erst einmal gelöscht, damit ich nicht durcheinander komme. Velen Dank für Deine Hilfe :-)

**Reply by Mersag** [reply]
Hello Paul,
 is it possible that we can meet for 5 minutes in Zoom call today, ab 13:30 ? Ia will be in bo-web-22-1 Room 5
